src/js/api/auth/logout.test.js

```diff
@@ -0,0 +1,27 @@
+import { logout } from "./logout";
+
+class LSMock {
+  constructor() {
+    this.storage = {
+      token: "token",
+      profile: `{"name":"god_gamer","email":"godgamer@noroff.no","banner":"","avatar":""}`,
+    };
+  }
+
+  getItem(key) {
+    return this.storage[key] || null;
+  }
+  removeItem(key) {
+    delete this.storage[key];
+  }
+}
+
+global.localStorage = new LSMock();
+
+describe("logout", () => {
+  it("removes saved token in browser", () => {
+    logout();
```

Comment:
You need to manually set the token in localStorage first otherwise this test will always pass